vsts/scripts/notifyBuildFailure.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT license.
+# --------------------------------------------------------------------------------------------
+
+set -e
+
+declare -r createBugFileContent='[{"op": "add","path": "/fields/System.Title","from": null,"value": "Build Failed-'$BUILD_DEFINITIONNAME'-'$BUILD_BUILDNUMBER'"}, \
+                                                                  {"op": "add", path": "/fields/System.AreaPath", "value": "DevDiv\Azure Developer Experience\Oryx"}, \
+                                                                  {"op": "add", "path": "/fields/Microsoft.DevDiv.OrgRank","value": "9999"}]'
+if [ "$BUILD_SOURCEBRANCH" == "refs/heads/master" ] && [ "$BUILD_REASON" == "Schedule" ] && \
```

Comment:
Do we want these conditions here since they are already being used in the job/step of a build definition?

Reply:
yeah I will remove them .. I added them when I was testing them inside container :)